src/+parameters/setTempParameters.m

```diff
@@ -3,9 +3,10 @@
         %Set temperature parameters for different landcover type.
     %}
     siteName = cellstr(siteName);
-    landcovers = unique(landcoverClass); % where landcoverClass is an array like ["forest", "forest", "forest", "shrubland", ...]
+    % where landcoverClass is an array like {"forest", "forest", "forest", "shrubland", ...}
+    landcovers = unique(landcoverClass);
     for ii = 1:length(landcovers)
-        [Vcmo, Tparam, m, Type, Rdparam, leafwidth] = landcover_variables(landcovers(ii));
+        [Vcmo, Tparam, m, Type, Rdparam, leafwidth] = landcover_variables(landcovers(ii), siteName);
         ScopeParameters.lcVcmo(ii) = Vcmo;
         ScopeParameters.lcTparam(ii) = Tparam;
```

Comment:
@Yunfei-Wang1993 this line is broken; `Tparam` is a 1x5 vector, while `lcTparam(ii)` is of size 1x1